core/src/apps/bitcoin/sign_tx/bitcoin.py

```diff
@@ -1,13 +1,12 @@
-from micropython import const
 from typing import TYPE_CHECKING
 
+from apps.common.writers import write_compact_size
+from micropython import const
```

Comment:
this is still not right. this change (and others in similar vein) should not exist. it looks like your config changes do not get picked up by isort, or maybe the options are called something else in the new version, or...?

Reply:
It is [as you suspected](https://pycqa.github.io/isort/docs/configuration/options.html#known-third-party:~:text=%2D%2Dfuture-,Known%20Third%20Party,-Force%20isort%20to) the options need to be defined in `.isort.cfg` or `pyproject.toml` file. I'll go ahead and add it to  the `pyproject.toml` file . 

Should I create a `.isort.cfg` file instead ?